packages/airnode-node/src/evm/authorization/authorization-application.test.ts

```diff
@@ -33,7 +33,12 @@ describe('mergeAuthorizations', () => {
     const apiCall = fixtures.requests.buildApiCall({ id: '0xapiCallId' });
     const authorizationByRequestId = { '0xapiCallId': true };
     const [logs, res] = authorization.mergeAuthorizations([apiCall], authorizationByRequestId);
-    expect(logs).toEqual([]);
+    expect(logs).toEqual([
+      {
+        level: 'DEBUG',
+        message: 'Requester:requesterAddress is authorized to access Endpoint ID:endpointId for Request ID:0xapiCallId',
```

Comment:
It would also be nice debug level info which requester authorized the given call.

Reply:
We previously had no logging when a request was authorized, so even this is an upgrade 😅. I looked into adding the originating authorizer to the log, but that is not trivial to do based on how authorization fetching currently occurs. I think it's doable, but outside the scope of this PR. Happy to open a separate issue though.

Reply:
https://github.com/api3dao/airnode/issues/1679